index.html

```diff
@@ -57,6 +57,132 @@ <h1 class="intro">I’m Selase 👋 Glad to see you!</h1>
+                    <p class="tag">javascript</p>
+                </div>
+                <button class="btn">See Project</button>
+            </article>
+
+            <article class="card">
+                <img src="./images/stories-2.png" alt="" aria-hidden="true">
+                <h3 class="title color-caption">Multi-Post
+                    Stories</h3>
+                <div class="profile">
+                    <div class="client">Canopy</div>
+                    <div class="role color-gray">Back End Dev</div>
+                    <div class="year color-gray">2015</div>
+                </div>
+                <p class="details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="tags">
+                    <p class="tag">html</p>
+                    <p class="tag">css</p>
+                    <p class="tag">javascript</p>
+                </div>
+                <button class="btn">See Project</button>
+            </article>
+        </section>
+
+        <section class="about-self wrapper">
+            <h2 class="sr-only">About Section</h2>
+            <h3 class="intro about-intro color-caption">About
+                Myself</h3>
+            <p class="description">
+                I’m a software developer! I can help you build a product , feature or website. Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                to contact me.
+            </p>
+            <p class="social-media">Let's connect</p>
+            <ul class="socials socials-about">
+                <li class="social-icon">
+                    <a href="#" aria-label="Github">
+                        <img src="./images/github.svg" alt="">
+                    </a>
+                </li>
+                <li class="social-icon">
+                    <a href="#" aria-label="Linkedin">
+                        <img src="./images/Linkedin.svg" alt="">
+                    </a>
+                </li>
+                <li class="social-icon">
+                    <a href="#" aria-label="AngelList">
+                        <img src="./images/AngelList.svg" alt="">
+                    </a>
+                </li>
+                <li class="social-icon">
+                    <a href="#" aria-label="twitter">
+                        <img src="./images/twitter.svg" alt="">
+                    </a>
+                </li>
+                <li class="social-icon">
+                    <a href="#" aria-label="medium">
+                        <img src="./images/medium.svg" alt="">
+                    </a>
+                </li>
+            </ul>
+            <button class="btn btn-resume">Get my resume</button>
+            <ul class="skill-list">
+                <li class="skill">Languages<span><img src="./images/chevron.png" alt=""></span></li>
```

Comment:
Kindly add the technologies part to your design, you don't have to make any functionality, just add the HTML and CSS for it.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/153520326-ae2554c1-0857-4646-ae60-bccc5a9a67f9.png) | ![image](https://user-images.githubusercontent.com/35535130/153520395-d281c18d-0388-4da5-aba2-78163f6e7635.png) |